xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/changesdoc.vm

```diff
@@ -2,6 +2,26 @@
 ##
 ## Preamble: which versions are being compared
 ##
+
+#if ("$!previousOrigdocVersion" != '')
+  #set ($changesLink = ${xwiki.getURL($tdoc, 'view', "viewer=changes&rev1=${previousOrigdocVersion}&rev2=${rev2}")})
+  #set ($previousOrigdocLink = "<a class='changes-arrow' href='$changesLink'>&lt;</a>")
```

Comment:
Just realized that we are missing a `title` for these links, to describe the action when the user hovers the link with the mouse.

They could be generic for the from/to versions, so that we need only 2 translation keys, i.e. "Previous version" and "Next version".